bezsmertnyi-stanislav/Readme.MD

```diff
@@ -0,0 +1,68 @@
+
+## Installation
+
+* Download the Flashscore Live Scores app from [AppStore](https://apps.apple.com/app/id766443283?mt=8) or [Google Play Store](https://play.google.com/store/apps/details?id=eu.livesport.FlashScore_com)
+* Install the app. You'll find the downloaded app in "Notifications" or in the "Download" folder. Click on it, choose "Install" and confirm
+* Launch the app
+
+## Features
+
+* COVERAGE: Discover live scores for football, tennis, basketball, and over 30 other sports (including rugby, cricket, and more) within our application. We monitor 6000+ competitions from around the world, with a dedicated focus on over 1000+ football tournaments alone!
+
+* SPEED: You will immidiatly receive updates, synchronized with the live audience, about scored goals, red cards, a set or period concludes.
+
+* FAVOURITE TEAMS AND MATCHES: You can follow your favourite matches, teams and competitions.
+
+* NOTIFICATIONS AND ALERTS: You can select your preferred matches and await the timely notifications on your mobile device.
+
+* SYNCHRONIZATION: As a registered user, your personalized live scores data will be accessible across all your devices.
+
+* REAL-TIME COMMENTARY: Stay up-to-date with our live text commentary.
+
+* LINE-UPS AND HEAD-TO-HEAD RECORDS: We provide advanced access to line-ups and the historical records, enabling you to find out the past encounters between both teams.
+
+## Usage
+
+* ### How to register
+
+    * Go to settings by tapping on cogwheel icon
+    * Click on button "Log in" in user section
+    * On the registration screen choose the way how you want to register: with email, with facebook, with google
+    * If you choose with facebook or with google, just choose your account on facebook or gmail
+    * If you choose with email, you need to input email and come up with password
+
+* ### How to choose kind of sport
+
+    * On the home screen click on the name of sport
+    * In the list choose tap on the icon or tab representing the sport you wish to explore
+    * After selecting a specific sport, you will be redirected to a dedicated section that focuses only on that sport.
+ 
+* ### How to add team to favourites
+
+    * Find a team by clicking on search button and putting your team name
+    * In the list of result find your team and tap on star icon
+
+* ### Navigation bar
+    ![](https://i.imgur.com/p3Q9PGz.jpg)
+
+    * All Games: This button allows you to view all available games across different sports, leagues, and competitions
+    * Live: The Live button filters and displays only the games that are currently in progress
+    * Favourites: The Favorites button provides quick access to your selected favorite teams, matches, or competitions
+    * News: The News button directs you to a dedicated section where you can explore the latest sports news, articles, and updates
+    * Standings: The Standings button allows you to access the league standings or tables for different sports and competitions
```

Comment:
You can make the text shorter by removing unnecessary wordage. For example:
* All Games: View all available games across different sports, leagues, and competitions.
* Live: Filter games to only see the ones that are in progress.
* Favorites: Get quick access to your favorite teams, matches, and competitions.
* News: Explore the latest sports news and updates.
* Standings: Access the league standings or tables for different sports and competitions.